### DISCUSSION TITLE
Free Electricity Sessions

### DISCUSSION THREAD

**Discussion by MHKnowles** [BottlecapDave/HomeAssistant-OctopusEnergy#974] [General]
Hi all, I was emailed by Octopus about this yesterday where they're going to give people who sign up an hour's free electricity today at 1pm.
Have we had any additional sensors exposed by their API to cover this yet?

**Comment by BottlecapDave** [comment]
Hello. This has been raised at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/973. Please follow this issue for updates on when the data is available.
